client/my-sites/email/form/mailboxes/validators.ts

```diff
@@ -255,28 +255,39 @@ class PasswordValidator extends BaseValidator< string > {
 
 class ExistingMailboxNamesValidator extends BaseValidator< string > {
 	private readonly existingMailboxNames: string[];
+	private readonly domainName: string;
 
-	constructor( existingMailboxNames: string[] ) {
+	constructor( domainName: string, existingMailboxNames: string[] ) {
 		super();
+		this.domainName = domainName;
 		this.existingMailboxNames = existingMailboxNames;
 	}
 
-	static getExistingMailboxError(): FieldError {
-		return i18n.translate( 'Please use unique mailboxes.' );
+	static getExistingMailboxError( domainName: string, existingMailbox: string ): FieldError {
+		return i18n.translate(
+			'Please use unique email addresses. {{strong}}%(emailAddress)s{{/strong}} already exists in your account',
```

Comment:
Let's not forget the trailing dot:

```suggestion
			'Please use unique email addresses. {{strong}}%(emailAddress)s{{/strong}} already exists in your account.',
```

Reply:
Actually the current validation shows an error message coming from the backend:

![image](https://user-images.githubusercontent.com/594356/173542156-a72d0945-2e1b-41f8-9c63-a5ba41bc5db2.png)

Can't we keep that particular message? Why are we defining a message in Calypso, especially since we're planning on validating those fields in the backend in the future (see 1165571745647867-as-1199621378640590 in Asana)?

Reply:
That backend validation still exists and works when the user submits the form. 

- The existing mailboxes validation is different in that it only checks for existing names
- The availability API which is actually proxied to the provider checks for other things apart from existing mailboxes, as you may also have groups and aliases in the same namespace.
- This existing mailboxes validation is in place mostly for Google as an alternative availability API is not available yet.